packages/workflows-shared/src/binding.ts

```diff
@@ -14,12 +14,9 @@ type Env = {
 // this.env.WORKFLOW is WorkflowBinding
 export class WorkflowBinding extends WorkerEntrypoint<Env> implements Workflow {
 	public async create({
-		id,
-		params,
+		id = crypto.randomUUID(),
+		params = {},
 	}: WorkflowInstanceCreateOptions): Promise<WorkflowInstance> {
```

Comment:
```suggestion
		id = crypto.randomUUID(),
		params = {},
	}: WorkflowInstanceCreateOptions = {}): Promise<WorkflowInstance> {
```

I'm pretty sure we're still missing this?

Reply:
well catched!